packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1386,25 +1433,65 @@ export const components: ThemeOptions["components"] = {
   MuiRadio: {
     defaultProps: {
       size: "small",
+      icon: <></>,
+      checkedIcon: <></>,
     },
     styleOverrides: {
       root: ({ theme }) => ({
-        "&:hover": {
+        width: theme.spacing(4),
+        height: theme.spacing(4),
+        borderRadius: theme.spacing(4),
+        borderWidth: "1px",
+        borderStyle: "solid",
+        borderColor: theme.palette.grey[500],
+
+        ".MuiFormControlLabel-root:hover > &": {
+          backgroundColor: "transparent",
+          borderColor: theme.palette.grey[900],
+        },
+        ".Mui-error:hover > &": {
           backgroundColor: "transparent",
+          borderColor: theme.palette.error.dark,
         },
         padding: 0,
         ".Mui-error > &": {
-          color: theme.palette.error.main,
-          "&:hover": {
-            color: theme.palette.error.dark,
+          borderColor: theme.palette.error.main,
+          "&::before": {
+            backgroundColor: theme.palette.error.main,
           },
         },
         "&.Mui-focusVisible": {
+          borderColor: theme.palette.grey[900],
           outlineColor: theme.palette.primary.main,
-          outlineOffset: 0,
+          outlineOffset: "2px",
           outlineStyle: "solid",
           outlineWidth: "2px",
         },
+        "&.Mui-checked": {
+          position: "relative",
+
+          "&::before": {
+            content: "''",
+            width: theme.spacing(2),
+            height: theme.spacing(2),
+            borderRadius: theme.spacing(2),
+            backgroundColor: theme.palette.primary.main,
+            position: "absolute",
+            top: theme.spacing(0.5),
```

Comment:
Looks like these declarations aren't making it through to the build because `spacing(0.5)` isn't a valid value.

Might be safe to simply remove them.